driver/mysql/mysql_test.go

```diff
@@ -29,6 +29,22 @@ func TestMigrate(t *testing.T) {
 		t.Fatal(err)
 	}
 
+	migrate(t, driverUrl)
+
+	if _, err := connection.Exec(`DROP TABLE IF EXISTS yolo, yolo1, ` + tableName); err != nil {
```

Comment:
It's in the migrations inside migrate. The migrations create a "yolo" table that has to be dropped so the test can run again.